edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/EdgeHubConfig.cs

```diff
@@ -5,13 +5,24 @@ namespace Microsoft.Azure.Devices.Edge.Hub.Core.Config
     using System.Collections.Generic;
     using Microsoft.Azure.Devices.Edge.Util;
 
+    /// <summary>
+    /// Domain object that represents EdgeHub configuration.
+    ///
+    /// This object is being eventually constructed from the EdgeHub twin's desired properties.
+    /// See <see cref="EdgeHubDesiredProperties"/> for DTO.
+    /// </summary>
     public class EdgeHubConfig : IEquatable<EdgeHubConfig>
     {
-        public EdgeHubConfig(string schemaVersion, IReadOnlyDictionary<string, RouteConfig> routes, StoreAndForwardConfiguration storeAndForwardConfiguration)
+        public EdgeHubConfig(
+            string schemaVersion,
```

Comment:
nit: should use System.Version type

Reply:
since this is nit I'll skip that. Don't want to add more to this PR. Maybe in the next one.